programs/drift/src/state/oracle_map.rs

```diff
@@ -42,8 +42,16 @@ impl<'a> OracleMap<'a> {
             .clone())
     }
 
+    fn should_get_quote_asset_price_data(&self, pubkey: &Pubkey) -> bool {
+        #[cfg(feature = "mainnet-beta")]
+        return (self.slot < 185740418 && pubkey == &usdc_oracle::id())
```

Comment:
what are you going to set this too?

Reply:
was just going to look at the current slot and add a weeks worth of time